docs/design/create-cr-by-operandrequest.md

```diff
@@ -0,0 +1,62 @@
+# Create Custom Resource by OperandRequest
+
+Using OperandConfig can apply a default custom resource according to the alm-example in the CSV, which provides convenience to users using a template to customize their own custom resource.
+
+However, while it provides convenience, it also creates some limitations:
+
+- Users can't create multiple custom resources for the same CustomResourceDefinition.
+- Users can't create the custom resource in a different namespace from the operator.
+- Users have to update the OperandConfig to customize the custom resource.
+
+## How to create Custom Resource by OperandRequest
+
+```yaml
+apiVersion: operator.ibm.com/v1alpha1
+kind: OperandRequest
+metadata:
+  name: db2-instance1
+  namespace: my-service
+spec:
+  requests:
+  - registry: common-service
+    namespace: ibm-common-services
+    operands:
+    - name: ibm-db2-operator
+      kind: db2
+      apiVersion: operator.ibm.com/v1alpha1
+      instanceName: db2-instance1
+      spec:
+        replicas: 3
+    - name: ibm-db2-operator
+      kind: db2
+      apiVersion: operator.ibm.com/v1alpha1
+      instanceName: db2-instance2
+      spec:
+        replicas: 3
+```
+
+The above `OperandRequest` will create two `db2` custom resources in the `my-service` namespace and the `db2` operator will be created in the namespace specified in the OperandRegistry `ibm-common-services/common-services`.
```

Comment:
unclear where `db2` operator will be created. Is it deployed in `ibm-common-services` or `common-services`?

Reply:
Let me make it clear. I mean it the namespace info is in the OperandRegistry `common-service` in the `ibm-common-services`, Let me update it to make it clear.